sos/plugins/__init__.py

```diff
@@ -1256,6 +1256,10 @@ def _collect_cmd_output(self, cmd, suggest_filename=None,
                         poller=self.check_timeout
                     )
             self._log_debug("could not run '%s': command not found" % cmd)
```

Comment:
Shouldnt we increase verbosity to `info`, we we (intentionally and correctly) dont create blank files? Like we do when we skip collecting a cmd output due to predicate - default execution of `sosreport` should contain a log about it.

Reply:
I don't think so, personally.

It's not uncommon for plugins to call the same binary multiple times with different options so we could quickly end up spamming users unnecessarily. The predicate warnings have, so far, generally been a low number of reports since a lot of their use is around gating commands that then iterate over some discovered list (like in the docker plugin).

If we changed this to `info`, I think we'd blast to much information at the end user. For the party that ultimately looks at the archive, if there are questions about missing command output they'd have to look at the logs anyways, so I don't think we gain much by increasing the log level.